apps/docs/src/packages/vitepress-theme.md

```diff
@@ -62,3 +62,38 @@ export default OnyxTheme;
 ```
 
 :::
+
+## Utilities
+
+This package also includes some VitePress utilities that you can use on-demand.
+
+### Breakpoint SCSS mixin
+
+Applies CSS only to the given VitePress breakpoint.
+
+::: details Available VitePress breakpoints
+
+| Breakpoint name | Value    |
+| --------------- | -------- |
+| xs              | `375px`  |
+| s               | `640px`  |
+| m               | `768px`  |
+| l               | `960px`  |
+| xl              | `1280px` |
+| xxl             | `1440px` |
+
+:::
+
+```scss
+@use "@sit-onyx/vitepress-theme/mixins.scss";
+
+.some-class {
+  @include mixins.breakpoint(m, max) {
+    // your styles for m breakpoint and smaller
+  }
+
+  @include mixins.breakpoint(m, min) {
+    // your styles for m breakpoint and larger
```

Comment:
1) sure? or did smth get flipped?
```suggestion
  @include mixins.breakpoint(m, max) {
    // your styles for m breakpoint and larger
  }

  @include mixins.breakpoint(m, min) {
    // your styles for m breakpoint and smaller
```

2) is "m" always included? so both would apply when you have exactly 768px?

Reply:
Yes because its `min-width: 768px` and `max-width: 768px`.
Should I add an optional third parameter `offset` so you can do this if you want to use both:
```scss
  @include mixins.breakpoint(m, max) {
    // your styles for m breakpoint and smaller
  }

  @include mixins.breakpoint(m, min, -1) {
    // your styles for m breakpoint and larger
  }
```

Reply:
either that or implicitly do this always on either min or max
+ docs

so that you get a consistent experience

Reply:
I'd prefer the offset parameter since I can imagine that it should be either applied to min or max, depending on the use case :)